src/background/communication.js

```diff
@@ -56,7 +56,7 @@ async function send(protocol) {
   await updateSentAt();
 }
 
-(async function setupStats() {
```

Comment:
What's the reason to remove the function name. Having a name helps to understand stacktraces.

Reply:
Leftover, i had to remove upcoming changes, tomorrow there will be no function here ;) so i think it is not worth to get it back here for a moment